app/service/rest.py

```diff
@@ -118,7 +118,10 @@ def handle_integrity_error(exc):
     """
     Handle integrity errors caused by the unique constraint on ix_organisation_name
     """
-    if 'services_name_key' or 'services_email_from_key' in str(exc):
+    if any(
+        'duplicate key value violates unique constraint "{}"'.format(constraint) in str(exc)
+        for constraint in {'services_name_key', 'services_email_from_key'}
```

Comment:
Why is this a set?

Reply:
i'm not super opinionated but i think its a reasonable habit to get into to use sets rather than lists for unordered distinct sequences

haven't really thought too much about it in this case though 🤷‍♂️